### DISCUSSION TITLE
W11 control center opens on top (not above) of W10 taskbar

### DISCUSSION THREAD

**Discussion by Rageplay** [valinet/ExplorerPatcher#1153] [General]
Hi,

Thanks for ExplorerPatcher, it makes W11 worthwile.
I like to use the W10 taskbar but since the first 2023 build (after the update glitch) the Control Center appears on top (not above) the Windows 10 taskbar.

Here's a picture of what I mean (excuse my non-screenshot photo) :
![photo_2023-03-20_18-14-54](https://user-images.githubusercontent.com/17230380/226420584-c238b692-c660-4c53-b3ee-5f116383b3b3.jpg)

As I click the control center it appears on top of the taskbar, covering more then half of it.
I can remember it working correctly on older W11 and ExplorerPatcher version.

Info:
Win11 Enterprise 22H2 OS Build 22621.1344
Open-Shell and 7+ Taskbar Tweaker are installed as well
ExplorerPatcher version 22621.1413.54.3

I've searched and didn't find this issue somewhere else; perhaps under different tags?

Hopefully this can be solved (perhaps on my end) as this is the only thing preveting me using explorerpatcher on my main PC.

Thanks.

**Comment by valinet** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1144

Also, solved on the latest OS build with latest EP: https://github.com/valinet/ExplorerPatcher/issues/1093

**Reply by Rageplay** [reply]
I've updated Windows 11 and it seems to work correct now. Thanks.

**Reply by valinet** [reply]
Okay then. Yeah, the fix is specifically made available only on OS build 22621.1413+, even though it may have worked/been required on 1344, but since that was obsoleted by the time the fix was released and I did not have a VM with that version to test against, I just played it safe and decided to ignore it, expecting people will upgrade anyway to the non-preview build.
